ui/src/utils/AudioRecorder.js

```diff
@@ -0,0 +1,47 @@
+export class AudioRecorder {
```

Comment:
I believe this version of the file is out of date with Master

Reply:
Looks like it got moved to `utils` folder? This is showing as a new file diff.

Reply:
1. this branch is a little old (i'll pull new versions from master) 2.  I moved it to a utils folder, but I'll move it back for now so the merge conflicts can happen and they can be addressed